packages/kolibri-tools/lib/cli.js

```diff
@@ -224,6 +226,8 @@ program
     list,
     []
   )
+  .option('--kds', 'Flag to use local kds', false)
```

Comment:
I think my only comment is that having two flags seems like one too many! I would prefer to avoid the default behaviour and just require the explicit path to KDS.

The other reason this is preferable is that in situations where `kolibri-tools` is being used outside of the Kolibri repo, the exact nested path that is encoded in the default may not match.